questions.txt

```diff
@@ -19,3 +19,94 @@ correct: $C\mathrm{e}^t$
 incorrect: $C\sin{t}$
 incorrect: $C\cos{t}$
 incorrect: $C\log{t}$
+
+------------------ Question
+
+type: TF
+prompt: The matrix, 
+\$A = \begin{bmatrix}
+\1 & 2 & 3\\
+\2 & 9 & 2\\
+\4 & 8 & 12\end{bmatrix}$, has rank 2.
+answer: true
+
+------------------ Question
+
+type: ESS
+prompt: Explain why the matrix, 
+\$A = \begin{bmatrix}
+\1 & 2 & 3\\
+\2 & 9 & 2\\
+\4 & 8 & 12\end{bmatrix}$, will have an eigenvalue equal to 0.
+example: The rank of this matrix is 2, meaning the nullspace has dimension 1.
+\The number of zero eigenvalues is directly linked to the dimensionality of the
+nullspace as this describes how many eigenvectors will be reduced to the zero
+vector.
+
+------------------ Question
+
+type: ORD
+prompt: $X$ follows a continuous distribution and has probability density
+\function
+>$\begin{array}{l}f_X(x) = \frac{1}{2}x, \quad 0 \leq X \leq 2\\
+\\quad\quad \quad = 0, \quad \mathrm{elsewhere.}\end{array}$
+>Order the following probability measures with the largest at the top.
+answer: The mode of $X$
+answer: The median of $X$
+answer: The mean of $X$
+
+------------------ Question
+
+type: MAT
+prompt: For the complex number, $z = 2+3j$, match the following expressions to
+\their corresponding value.
```

Comment:
Using `\` like this is confusing. Normally `\` is used as a line continuation character at the end of the line e.g.:
```
this line continues \
below.
```
Also can we make actual line breaks using <br> or something?